com.ktechpit.ultimate-media-downloader.yaml

```diff
@@ -0,0 +1,225 @@
+app-id: com.ktechpit.ultimate-media-downloader
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: ultimate-media-downloader
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --device=dri
+  - --socket=pulseaudio
+  - --env=XDG_CURRENT_DESKTOP=kde
```

Comment:
why? This is supposed to reflect what the user is running.

Reply:
I thought since I was using KDE runtime. Thanks anyway.